src/pages/html/html-entities/index.md

```diff
@@ -3,13 +3,25 @@ title: HTML Entities
 ---
 ## HTML Entities
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/html-entities/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+In HTML characters like ```<```, ```>``` has special meaning. For example, if you use ```<``` character, browser will parse it as a beginning of an HTML element tag. Instead you need to use HTML entity&mdash;a string beginning with ```&``` and ending with ```;```. And for any special symbol there is an HTML entity.
```

Comment:
Fix grammatical error "For example, if you use  ```<```  character, browser will parse" into "For example, if you use  **the** ```<```  character, **the** browser will parse"